infrastructure/metadata/infrastructure/testdriver/set_permission.https.html.ini

```diff
@@ -1,8 +1,8 @@
 [set_permission.https.html]
```

Comment:
Does removing this file cause Firefox and/or Safari to break? I thought everyone implemented set_permission at this point.

Reply:
If that's related to https://wpt.fyi/results/webdriver/tests/permissions?label=master&label=experimental&product=chrome&product=firefox&product=safari it's not implemented in Firefox and Safari still.

Reply:
It's related to https://wpt.fyi/results/infrastructure/testdriver/set_permission.https.html?label=master&label=experimental&product=chrome&product=firefox&product=safari, but in any case it does look like the root cause is the same, so thanks for pointing it out :-)